src/Controller/Admin/AdminIdeaController.php

```diff
@@ -106,4 +106,18 @@ public function finalizeAction(Idea $idea, ObjectManager $manager): Response
 
         return $this->redirectToRoute('admin_app_ideasworkshop_idea_list');
     }
+
+    /**
+     * @Route("/{id}/extend", name="app_admin_idea_extend", methods={"GET"})
+     */
+    public function extendAction(Idea $idea, ObjectManager $manager): Response
+    {
+        $idea->publish();
```

Comment:
Admin can infinitly extends an pending or finalized idea, and should not increment the `$extendsCount`

Reply:
what about adding the status check here too?